lib/auth.ts

```diff
@@ -111,6 +126,81 @@ export class Auth {
     return token;
   };
 
+  /**
+   * Perform a request against Kontist REST API
+   */
+  private request = async (path: string, method: HttpMethod, body?: string) => {
```

Comment:
I was contemplating extracting this to another class / module, what do you think? I also couldn't think of a proper name for this extracted class while implementing so I left it here for now.

Reply:
I would say it's fine to keep it here for now